tests/sentry/integrations/vercel/test_integration.py

```diff
@@ -205,7 +205,10 @@ def test_update_organization_config(self):
             responses.GET,
             "https://api.vercel.com/v5/projects/%s/env"
             % "Qme9NXBpguaRxcXssZ1NWHVaM98MAL6PHDXUs1jPrgiM8H",
-            json={"envs": []},
+            json={
+                "envs": [],
+                "pagination": {"count": 10},
```

Comment:
do we actually get the pagination if we don't have results?

Reply:
like is pagination in the response if there are no env vars? I'll check